packages/frontend/src/app/routing/app-routing.module.ts

```diff
@@ -28,8 +28,23 @@ const routes: Routes = [
   {path: '', component: HomeComponent},
   {path: 'license', component: LicenseComponent},
   {path: 'org/:org/:repo/:build', component: BuildComponent},
-  {path: RouteProvider.routes.repo.path, component: RepositoryComponent},
```

Comment:
The breadcrumb path is reflected in the URL as well, i.e.,

We'd be at the URL: `/org/GoogleCloudPlatform/flaky-dev`, and have the breadcrumb `GoogleCloudPlatform -> flaky-dev`.